pkg/kubectl/cmd/label.go

```diff
@@ -44,6 +45,9 @@ $ kubectl label --overwrite pods foo status=unhealthy
 // Update all pods in the namespace
 $ kubectl label pods --all status=unhealthy
 
+// Update a pod using the data in pod.json
```

Comment:
Please explain that this just gets the identities of the resources to get from the file(s).
Update comments from `//` to `#` also.